superset-frontend/src/dashboard/stylesheets/dashboard.less

```diff
@@ -19,6 +19,7 @@
 /* header has mysterious extra margin */
 header.top {
   margin-bottom: 2px;
+  z-index: 2;
```

Comment:
This works, but we should probably tackle the underlying issue that causes this.
The culprit seems to be the right nav, which is fixed

Reply:
This issue is happened because that dashboard build side pane is over on top menu.
![Screen Shot 2022-04-21 at 1 00 38 PM](https://user-images.githubusercontent.com/47900232/164513350-32bbcb16-4fc1-4399-b864-62d438a1d2eb.png)

I think that as a result, this is not limited in right menu.